otsserver/stamper.py

```diff
@@ -350,26 +345,28 @@ def __do_bitcoin(self):
                 # have been mined, and are waiting for confirmations.
                 self.txs_waiting_for_confirmation[block_height] = mined_tx
 
-                # Erasing all unconfirmed txs if the transaction was mine
-                if mined_tx.tx.GetTxid() in self.mines:
-                    self.unconfirmed_txs.clear()
-                    self.mines.clear()
+                # Erasing all unconfirmed txs, they are all conflicting with each other
```

Comment:
Better to word that as "Erase all unconfirmed txs, as they all conflict with each other"

Similar grammar principle as what I said re: git commits.